templates/page.html

```diff
@@ -191,12 +202,17 @@
       <article>
         <div class="post-header">
           <h1>{{title}}</h1>
-          <p>
+          <div>
             <span class="sr-only">Published on</span>
-            <time datetime="{{publishDate}}" class="dt-published"
-              >{{publishDate}}</time
-            >
-          </p>
+            <time datetime="{{publishDate}}" class="dt-published">{{publishDate}}</time>
+            {{#categories.length}} in <span class="sr-only">the following categories:</span>
+            <ul class="categories" role="list">
+              {{#categories}}
+              <li><a href="/categories/{{name}}">{{name}}</a></li>
+              {{/categories}}
+            </ul>
+            {{/categories.length}}
+          </div>
```

Comment:
This is how category links are displayed on notes (styles above in the same file)